lib/src/sai_redis_fdb.cpp

```diff
@@ -16,42 +16,6 @@ static sai_status_t redis_flush_fdb_entries(
 REDIS_GENERIC_QUAD_ENTRY(FDB_ENTRY,fdb_entry);
 REDIS_BULK_QUAD_ENTRY(FDB_ENTRY,fdb_entry);
 
-// TODO remove when test corrected (SAI pointer must be advanced for this)
```

Comment:
Is this file changes needed part of first PR

Reply:
No. I'll remove this change from this PR